.travis.yml

```diff
@@ -15,6 +15,7 @@ notifications:
 templates:
   django21: &django21 DJANGO_VERSION=2.1.*
   django22: &django22 DJANGO_VERSION=2.2.*
+  django30: &django30 DJANGO_VERSION=3.0.*
```

Comment:
ERROR: No matching distribution found for Django==3.0.*, could you try 3.0a1 instead?